### DISCUSSION TITLE
Unique visitors are summed over the periods longer than a day

### DISCUSSION THREAD

**Discussion by olejandro** [plausible/analytics#1445] [Anything Else]
Currently the number of unique visitors shown on the dashboard over a period longer than a day is summed. However, as far as I understand, there is no way to tell the actual number of unique visitors in a period longer than one day. Would it be more accurate to e. g.:

- Show an average instead of the sum?
- Change "unique visitors" to "visitors" when a custom range longer than one day is chosen?

**Comment by metmarkosaric** [comment]
thanks @olejandro! yeah, Plausible is built with GDPR in mind and with GDPR it is not possible to have any long term identifiers without user consent. average would be a bit weird as we want to show the total number of unique visitors in the time period rather than the average daily unique visitors in the time period. visitors is not entirely correct either as the daily numbers do show "unique visitors" rather than "visitors" which would be total (we do plan to introduce "sessions" at some point which is more like "visitors" metric). i guess you can look at it similar to unique visitors in GA and other tools which may not be entirely correct either due to cookies being cleared/blocked, GDPR consent not being given etc

**Reply by olejandro** [reply]
Thanks @metmarkosaric! My main concern is that a user of the dashboard may not understand what the unique visitor count over a period longer than 1 day represents without reading how Plausible defines unique visitors. At the same time I realise that it is important not to overcrowd the UI. :-) Could something like an (i) next to Unique Visitours count which could say e. g. "sum of daily unique visitors" on hover over be a good way of ensure clarity?

**Reply by metmarkosaric** [reply]
perhaps if we can make it very tiny as normally those "i"'s ruin a UI and would become too prominent on a clean/minimal dashboard such as the one we have. we strive to be simple and easy to understand and in general there doesn't seem to be too many questions and doubts about what our metrics do so it's not something that's necessary to add. we make it very clear everywhere that we are privacy-first and don't use cookies etc so in general people understand the differences. in our testing, our way of tracking unique visitors is pretty close to the GA method. bigger differences in stats come from the GDPR consent and adblockers than from our definitions being different.

**Reply by olejandro** [reply]
True! I like the current PA UI very much, so definitely would not want to impact it negatively... Another (simpler?) idea could be to adjust the text that is displayed on hover over the unique visitor count: i.e. currently it says "x unique visitors", but maybe it could say "x total unique daily visitors"? Just an idea. I guess, my main concern is that someone who has no idea about analytics (and how different tools compare) can get a bit confused just by looking at the number (e.g. in case the dashboard is shared with them). It may not be all that important after all.

**Reply by metmarkosaric** [reply]
my guess is that saying "total unique daily visitors" will confuse more people and we will see more people contacting us asking for what that means. "unique visitors" is pretty straight forward and people are used to it from pretty much all the different tools

**Reply by olejandro** [reply]
@metmarkosaric, just to clarify, my suggestion was not to change what's written on the dashboard, but only what appears when one hovers over the number representing the total over a period longer than one day. This is based on an assumption that that's how a user will look for more information. Until very recently, based on what appears on the dashboard, my own impression was that PA actually tracks unique visitors across days. However after spending a bit of time reading through the docs I now know that's not the case because PA generates a daily changing identifier. However a visitor to a site with whom a PA dashboard is openly shared may get an impression that some kind of a persistent identefier is generated despite the stated GDPR compliance. It would be great if this could be avoided somehow. :-)

**Reply by metmarkosaric** [reply]
i understand, thanks! we'll keep this in mind and consider what can be done

**Comment by luxaritas** [comment]
Not only will stakeholders not familiar with this setup likely be confused, I can see this throwing me off even knowing about it. IMO, no data is better than misleading data - particularly in this case, I'm not convinced the data as presented is meaningful. That said, I think showing average daily unique users would be a valid and useful way to go.

NB, there may be use cases where someone may be working in a setting where strict GDPR compliance isn't necessary - in those cases it could potentially be useful to provide an opt-in setting (with relevant warnings) to specify how often the salt rotation happens (eg, weekly or monthly - I'd personally argue that eg weekly would not significantly hurt user privacy while allowing concrete useful insights for some classes of applications)

**Reply by luxaritas** [reply]
Considering the salt gets thrown away after some time, I wouldn't be surprised if many individuals would even be comfortable with specifying a custom user ID that would be hashed, which could allow for further increased accuracy (even "perfect" accuracy for authenticated users).

**Comment by pkl** [comment]
SUM OF UNIQUES_DAY is not a meaningful metric at all - it's not what anyone expect for "UNIQUE VISITORS" over a timespan.

**Comment by olejandro** [comment]
Thanks @metmarkosaric! Out of curiousity, how was the behaviour changed?
